XMonad/Util/ClickableWorkspaces.hs

```diff
@@ -32,26 +33,33 @@ import XMonad.Hooks.DynamicLog (xmobarAction, xmobarRaw, PP(..))
 -- > logHook = clickablePP xmobarPP { ... } >>= dynamicLogWithPP
 --
 -- * Requirements:
---   * wmctrl on system (in path)
+--   * @xdotool@ on system (in path)
 --   * "XMonad.Hooks.EwmhDesktops" for wmctrl support (see Hackage docs for setup)
```

Comment:
```suggestion
--   * "XMonad.Hooks.EwmhDesktops" for @xdotool@ support (see Hackage docs for setup)
```

Plus, does xdotool need emwh even?

Reply:
For set_desktop and windowactivate, it does. It uses the same clientmessages as wmctrl, just sets the source indication correctly, as explained in more detail in the commit message.